scripts/start_logging.sh

```diff
@@ -18,12 +29,14 @@ pipe_pane_ansifilter() {
 pipe_pane_sed_osx() {
 	# Warning, very complex regex ahead.
 	# Some characters below might not be visible from github web view.
-	local ansi_codes_osx="(\[([0-9]{1,3}((;[0-9]{1,3})*)?)?[m|K]|
|]0;[^]+|[[:space:]]+$)"
```

Comment:
This and the next change are not mine. git insists on marking them as diffs. Not sure what's going on there.